include/dosbox.h

```diff
@@ -55,8 +55,7 @@ void DOSBOX_SetNormalLoop();
 void DOSBOX_Init(void);
 
 class Config;
-using config_ptr_t = std::unique_ptr<Config>;
-extern config_ptr_t control;
+extern Config* control;
```

Comment:
Um, maybe out of scope, but could we call this `config` instead?

Reply:
Got it!